### ISSUE TITLE
Add the session cookie to TrackingProgression

### ISSUE THREAD

**Issue by JeffAbrahamson** [transport-nantes/tn_web#173]
Help us do matching on events, see if users return / attempt frequently or if they're all different.

**Comment by Shriukan33** [comment]
So i've played a bit with the cookies to see how it works. 
What do you suggest ? Creating a random string if none is present in the cookies and save it along the other information in the tracking model ? 
Should we set an expiry date ? Default is 14 days apparently 
Would a random 20 long alphanumerical string would do ?

**Comment by JeffAbrahamson** [comment]
The cookie name has to be identifiable, otherwise you can't recover it.  Perhaps something like "tn_nav".  Django just represents it as a (key, value) pair in a dict, roughly.  As is natural in python.

Perhaps use `make_timed_token()` with a value of 6 months.

**Comment by Shriukan33** [comment]
> The cookie name has to be identifiable, otherwise you can't recover it. Perhaps something like "tn_nav". Django just represents it as a (key, value) pair in a dict, roughly. As is natural in python.
> 
> Perhaps use `make_timed_token()` with a value of 6 months. That way you can check if it's legit using `token_valid()`.

Yes, the name has to be identifiable, I was thinking about the value of tn_nav :slightly_smiling_face: 
Cookies are key-value pairs. 
There is a built in function that set expiry date, it just defaults to 14 days but I'm pretty sure that can be modified. It also accept dates iirc.

**Comment by JeffAbrahamson** [comment]
uaparser -- awesome!  I'm not sure what info we'll want of that eventually, but not collecting data is the best way not to have data.  From memory, I bet that's just a couple lines of code, so well worth it!

Field length of 50 is fine, of course.  It's just about being more than needed.  To much is enough...
